src/solver/solve.jl

```diff
@@ -241,26 +241,20 @@ function check_conservation(sol)
             energy_radiation_input,
         ) / energy_total
 
-    mass_conservation =
-        (sum(sol.u[end].c.ρ) - sum(sol.u[1].c.ρ)) / sum(sol.u[1].c.ρ)
-
-    water_conservation = zero(Spaces.undertype(axes(sol.u[end].c.ρ)))
+    water_surface_change = zero(Spaces.undertype(axes(sol.u[end].c.ρ)))
+    water_total = sum(sol.u[1].c.ρq_tot)
```

Comment:
```suggestion
    water_total = sum(sol.u[end].c.ρq_tot)
```
This should fix the error here: https://buildkite.com/clima/climaatmos-ci/builds/19579#01907544-653d-482c-ae12-4c964b6778c9/200-485